makefile

```diff
@@ -423,31 +423,10 @@ docker_buildandpackage: docker_ok
 # Test targets        #
 #---------------------#
 
-ifndef NORACETEST
-ifeq (,$(findstring history_size,$(GORACE)))
-export GORACE:=$(GORACE) history_size=7
-endif
-ifeq (,$(findstring halt_on_error,$(GORACE)))
-export GORACE:=$(GORACE) halt_on_error=1
-endif
-ifeq (,$(findstring -race,$(GOTEST_FLAGS)))
-export GOTEST_FLAGS:=$(GOTEST_FLAGS) -race
-endif
-endif
```

Comment:
Keeping the flags in make will keep the test execution and the build execution more consistent, plus it allows someone to experiment with changing the default options in way that's consistent with how Jenkins would run the build.